pages/api/endorsements/mentees/mentee_id.js

```diff
@@ -1,6 +1,25 @@
 // PUT: /pages/api/endorsements/mentees/[mentee_id].js
+import { db } from '../../../../utils/api/firebase.config';
+import {
+  collection,
+  doc,
+  updateDoc,
+  increment
+} from "firebase/firestore";
 
-export default function updateMenteeEndorsements(req, res) {
+export default async function updateMenteeEndorsements(req, res) {
```

Comment:
I don't see any anywhere that filters out a PUT request. Could this cause issues if another request type is sent to this endpoint? IE would a GET request to this endpoint increment the endorsement count? If not then disregard this comment